l2geth/internal/ethapi/api.go

```diff
@@ -1723,6 +1724,9 @@ func (s *PublicTransactionPoolAPI) SendTransaction(ctx context.Context, args Sen
 // FillTransaction fills the defaults (nonce, gas, gasPrice) on a given unsigned transaction,
 // and returns it to the caller for further processing (signing + broadcast)
 func (s *PublicTransactionPoolAPI) FillTransaction(ctx context.Context, args SendTxArgs) (*SignTransactionResult, error) {
+	if vm.UsingOVM {
+		return nil, errors.New("Cannot fill transaction")
```

Comment:
Should have some kind of prefix e.g., "OVMError: Cannot fill transaction" or in some other way indicate this error is thrown to indicate it is due to the vm being wrong

Reply:
IMO just create a global error called: `var errOvmNotSupported = errors.New("OVMError: Endpoint disabled")`, maybe wrapping it with some additional context per method, although not necessary.